pkg/keystore/keystore.go

```diff
@@ -52,13 +53,20 @@ func New(config *Config) *Keystore {
 	return &Keystore{config}
 }
 
-var defaultConfig = &Config{
-	RootPath:         "/etc/rkt/trustedkeys/root.d",
-	PrefixPath:       "/etc/rkt/trustedkeys/prefix.d",
-	SystemRootPath:   "/usr/lib/rkt/trustedkeys/root.d",
-	SystemPrefixPath: "/usr/lib/rkt/trustedkeys/prefix.d",
+func NewConfig(systemPath, customPath string) *Config {
+	tk := "trustedkeys"
```

Comment:
since these are function-local I think it's less cognitive overhead when parsing to just hardcode here
```
	return &Config{
		RootPath:         filepath.Join(customPath, "trustedkeys", "root.d"),
		PrefixPath:       filepath.Join(customPath, "trustedkeys", "prefix.d"),
		SystemRootPath:   filepath.Join(systemPath, "trustedkeys", "root.d"),
		SystemPrefixPath: filepath.Join(systemPath, "trustedkeys", "prefix.d"),
```

Alternatively, split them out to package level consts if they might be useful elsewhere.